src/model/cloud-runner/providers/k8s/kubernetes-storage.ts

```
@@ -49,26 +49,47 @@ class KubernetesStorage {
     try {
       CloudRunnerLogger.log(`watch Until PVC Not Pending ${name} ${namespace}`);
-      CloudRunnerLogger.log(`${await this.getPVCPhase(kubeClient, name, namespace)}`);
+      const initialPhase = await this.getPVCPhase(kubeClient, name, namespace);
+      CloudRunnerLogger.log(`Initial PVC phase: ${initialPhase}`);
+      
+      // Wait until PVC is NOT Pending (i.e., Bound or Available)
       await waitUntil(
         async () => {
-          return (await this.getPVCPhase(kubeClient, name, namespace)) === 'Pending';
+          const phase = await this.getPVCPhase(kubeClient, name, namespace);
+          return phase !== 'Pending';
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)